test/e2e/common/node/ephemeral_containers.go

```diff
@@ -80,4 +86,88 @@ var _ = SIGDescribe("Ephemeral Containers [NodeConformance]", func() {
 		framework.ExpectNoError(err, "Failed to get logs for pod %q ephemeral container %q", e2epod.FormatPod(pod), ecName)
 		gomega.Expect(log).To(gomega.ContainSubstring("polo"))
 	})
+
+	ginkgo.It("should update the ephemeral containers in an existing pod", func(ctx context.Context) {
+		ginkgo.By("creating a target pod")
+		pod := podClient.CreateSync(ctx, &v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{Name: "ephemeral-containers-target-pod"},
+			Spec: v1.PodSpec{
+				Containers: []v1.Container{
+					{
+						Name:    "test-container-1",
+						Image:   imageutils.GetE2EImage(imageutils.BusyBox),
+						Command: []string{"/bin/sleep"},
+						Args:    []string{"10000"},
```

Comment:
is it intended to be an infinity? Maybe use `infinity`

Reply:
This part of the test was taken from the e2e test "will start an ephemeral container in an existing pod" in the same file. I'm not sure why that figure was used. As this test now covers all of the ephemeral container endpoints the current thought is the above test could be removed after this test has been promoted to conformance.